x-pack/plugins/security_solution/public/management/pages/response_actions/view/response_actions_list_page.test.tsx

```diff
@@ -137,7 +137,7 @@ describe('Response actions history page', () => {
     ({ history } = mockedContext);
     render = () => (renderResult = mockedContext.render(<ResponseActionsListPage />));
     reactTestingLibrary.act(() => {
-      history.push(`${MANAGEMENT_PATH}/response_actions`);
+      history.push(`${MANAGEMENT_PATH}/response_actions_history`);
```

Comment:
Changes here are just tech debt updates. All the other tests have been working fine as we only asser the `search` value and not the entire URL.